target-tests/README.md

```diff
@@ -0,0 +1,58 @@
+# Target Tests
+
+*Hardware in the loop (HIL) test suite for the
+[pa-spl](https://github.com/EngJay/pa-spl) driver.*
+
+## Running the Tests
+
+The HIL tests are configured to use a 0x32 or 0x33 version of the PCB Artists
```

Comment:
Again, are you trying to say that there are versions of the I2C SPL Module that have the I2C address physically set to either 0x32 or 0x33?

Reply:
That is not the address, it's the firmware version. The sensor has a VERSION register and there are currently 3+ versions. The published version for mine is 0x32 but it returns 0x33, so I'm assuming they released another version of the firmware but didn't update the docs.